src/compile/scale/component.ts

```diff
@@ -31,16 +31,41 @@ export class ScaleComponent extends Split<ScaleComponentProps> {
   }
 
   /**
-   * Whether the scale definitely includes zero in the domain
+   * Whether the scale definitely includes or not include zero in the domain
    */
-  public domainDefinitelyIncludesZero() {
-    if (this.get('zero') !== false) {
-      return true;
+  public domainHasZero(): 'definitely' | 'definitely-not' | 'maybe' {
```

Comment:
nit: I'm not super sure about the wording here in this union, but non blocking either way.

Reply:
yeah, i like it better than `boolean | null` where null is maybe though. Given it's internal, we can always change later.